### PR TITLE
Add new company - Brikl

### PR DESCRIPTION
**Pull request by Gnoyoyo**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [x] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: company-profiles/brikl.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+5 days work week with flexible hours and WFH policy.
+
+## Region
+
+North America, Asia
+
+## Company technologies
+
+Coalition is the leading metrics-driven digital agency. We specialize in SEO, PPC, email marketing, social media marketing, and web design & development.
```
Comment (KieranRobson) [inline company-profiles/brikl.md:21]: I think this may be incorrect as the company name is `Brikl` rather than `Coaltion`

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Hi @Gnoyoyo!

All looks great other than a quick clarification needed!

File: company-profiles/brikl.md (hunk 1 above)
Comment (Gnoyoyo) [reply company-profiles/brikl.md:21]: Oh my bad it's copy monster will change it right away!

File: README.md [hunk 2]
```diff
@@ -104,6 +104,7 @@ Name | Website | Region
 [Brainstorm Force](/company-profiles/brainstorm-force.md) | https://brainstormforce.com/ | Worldwide
 [Brave Investments](/company-profiles/brave-investments.md)  | https://www.crunchbase.com/organization/brave-investment | Brazil
 [Bright Funds](/company-profiles/bright-funds.md) | https://www.brightfunds.org | USA
+[Brikl](/company-profiles/brikl.md) | https://www.brightfunds.org | North America, Asia
```
Comment (KieranRobson) [inline README.md:107]: I believe the URL listed is also a copy monster 😆

Comment (Gnoyoyo) [reply README.md]: fix, Thanks!! it seem that PR before coffee is not a good idea.

**Review by KieranRobson — APPROVED** [review]
After the slaying of the copy monsters, this looks good to me!

File: company-profiles/brikl.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+5 days work week with flexible hours and WFH policy.
+
+## Region
+
+North America, Asia
+
+## Company technologies
+
+Brikl is the leading eCommerce MicroStore technology partner for custom and promotional businesses worldwide.
```
Comment (KieranRobson) [inline company-profiles/brikl.md:21]: One quick change I noticed, this section is usually where the technologies that the company uses goes, to make it eaisier for applicants to know whether they have the skills to apply for. 

Could you add some of the tech that the company uses, i.e Python, Node.js etc

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Final passthrough lead to me to more changes 😆

File: company-profiles/brikl.md (hunk 3 above)
Comment (Gnoyoyo) [reply company-profiles/brikl.md]: gotcha, and fixed

**Review by KieranRobson — APPROVED** [review]
Looks great again!

**Review by dougaitken — APPROVED** [review]

**Pull request merged**